### PR TITLE
Report time taken by analysis script

### PR DESCRIPTION
**Pull request by PhilMiller**

### CODE REVIEW

**Review by zingale — APPROVED** [review]

File: regtest.py [hunk 1]
```diff
@@ -1090,6 +1091,9 @@ def test_suite(argv):
                                 with open(outfile) as f:
                                     print(f.read())
 
+                        analysis_time = analysis_start_time - time.time()
+                        suite.log.log(f"Analysis time: {analyis_time:.3f} s")
+
```
Comment (zingale) [inline regtest.py:1096]: oh, I missed the pylint error -- can you do another PR fixing the spelling of `analyis_time` in the log statement?

**Comment by PhilMiller** [comment]
Oops, yeah, I can do that. Unless you want to just make the quick fix. I'm
not concerned about attribution it whatever.

**Comment by zingale** [comment]
okay, I'll just push a fix

**Comment by PhilMiller** [comment]
I realized that I also got the direction of the subtraction wrong. This is what I get for hacked-together changes without testing them.

**Comment by ax3l** [comment]
Jup, just saw as well that your analysis times are now negative :)
